html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>html-block-and-inline</title>
+</head>
+
+<body>
+  <h1>Blue Americana Snuggie&reg; Blanket</h1>
+  <h2>by SnuggieStore</h2>
+  <p>Price: <s>&dollar;19.99</s> &dollar;14.99</p>
+  <p>Part of our new Americana collection, this <i>beautiful</i> blue is already a customer favorite!</p>
+  <p>Enjoy complete <b>Freedom of Movement!</b></p>
```

Comment:
Same here, it is best to use `<strong>` over `<b>` for the same reason mentioned above.